### PR TITLE
Chef 13 compatibility fixes

### PR DESCRIPTION
**Pull request by jcorme**

### CODE REVIEW

**Comment by jcorme** [comment]
[Chef 13: yum-osuosl](https://trello.com/c/8xZkKUOJ/961-chef-13-yum-osuosl)

File: Rakefile [hunk 1]
```diff
@@ -110,18 +110,19 @@ task snakeoil: snakeoil_file_path
...
 
 desc 'Run FoodCritic (lint) tests'
 task :lint do
-    run_command('foodcritic --epic-fail any .')
+  run_command('foodcritic --epic-fail any .')
 end
 
 desc 'Run RSpec (unit) tests'
 task :unit do
-    run_command('rspec')
```
Comment (ramereth) [inline Rakefile]: Please add ``rm -f Berksfile.lock`` here

**Comment by jcorme** [comment]
!bump major *

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a major-level version bump to v1.0.0.  Have a nice day!

**Pull request merged**